rewrite-java-core/src/test/java/BinaryDataTest.java

```diff
@@ -0,0 +1,92 @@
+import com.azure.recipes.v2recipes.TypeReferenceRecipe;
```

Comment:
These tests should also be in the root package  - `com.azure.recipes.core.v2`. (See my comment above about changing the root package name)

Reply:
Changed package name in commit [492daeb](https://github.com/Azure/azsdk-java-rewrite-recipes/pull/14/commits/492daeb48360470829cc54218911b9b3e1758ace)